doc/man3/SSL_CONF_cmd.pod

```diff
@@ -161,9 +161,21 @@ where applicable (e.g. B<X25519>, B<ffdhe2048>) or an OpenSSL OID name
 (e.g. B<prime256v1>). Group names are case sensitive. The list should be
 in order of preference with the most preferred group first.
 
-Currently supported groups for B<TLSv1.3> are B<P-256>, B<P-384>, B<P-521>,
-B<X25519>, B<X448>, B<ffdhe2048>, B<ffdhe3072>, B<ffdhe4096>, B<ffdhe6144>,
-B<ffdhe8192>.
+The supported groups for B<TLSv1.3> include:
+B<P-256>,
+B<P-384>,
+B<P-521>,
+B<X25519>,
+B<X448>,
+B<ffdhe2048>,
+B<ffdhe3072>,
+B<ffdhe4096>,
+B<ffdhe6144>,
+B<ffdhe8192>,
+B<MLKEM512>,
+B<MLKEM768>,
+and
+B<MLKEM1024>.
```

Comment:
Should we add a TODO for the hybrid versions?

Reply:
Once the base ML-KEM referees are present, it'll be obvious where the hybrids need to go, so I think this is not a priority.